module-cheatsheets/scRNA-seq-advanced-cheatsheet.md

```diff
@@ -0,0 +1,115 @@
+
+
+### `batchelor` and `harmony`
+
+Read the `batchelor` package documentation [**here**](https://rdrr.io/cran/batchelor/), and a vignette on its usage [**here**](https://rdrr.io/bioc/batchelor/f/vignettes/correction.Rmd).
+
+Read the `harmony` package documentation [**here**](https://rdrr.io/github/immunogenomics/harmony/man/harmony.html), and a vignette on its usage [**here**](https://cran.r-project.org/web//packages/harmony/vignettes/quickstart.html).
+
+| Library/Package      | Piece of Code      | What it's called    | What it does  |
+|----------------------|----------------------------|--------------------------------------------|--------------------------------------------------------------|
+| `batchelor`| [`MultiBatchPCA()`](https://rdrr.io/bioc/batchelor/man/multiBatchPCA.html)| Multi-batch PCA | Perform PCA across multiple gene expression matrices while accounting for batch differences |
+| `batchelor`| [`fastMNN()`](https://rdrr.io/bioc/batchelor/man/fastMNN.html)| Fast mutual nearest neighbors correction | Perform integration with mutual nearest neighbors using the `fastMNN` algorithm |
+| `harmony`| [`HarmonyMatrix()`](https://rdrr.io/github/immunogenomics/harmony/man/HarmonyMatrix.html)| Main Harmony interface | Perform integration with `harmony` |
+
+
+### `pheatmap` and `EnhancedVolcano`
+Read the `pheatmap` package documentation [**here**](https://rdrr.io/cran/pheatmap/).
+
+Read the `EnhancedVolcano` package documentation [**here**](https://rdrr.io/bioc/EnhancedVolcano/), and a vignette on its usage [**here**](https://bioconductor.org/packages/3.16/bioc/vignettes/EnhancedVolcano/inst/doc/EnhancedVolcano.html).
+
+| Library/Package      | Piece of Code      | What it's called    | What it does  |
+|----------------------|----------------------------|--------------------------------------------|--------------------------------------------------------------|
+| `pheatmap`| [`pheatmap()`](https://rdrr.io/cran/pheatmap/man/pheatmap.html)| Pretty heatmap | Plot a (pretty!) clustered heatmap |
+| `EnhancedVolcano`| [`EnhancedVolcano()`](https://rdrr.io/bioc/EnhancedVolcano/man/EnhancedVolcano.html)| Enhanced volcano | Plot a volcano plot to visualize differential expression analysis results |
+
+
+### Tidyverse functions
+
+Read the full documentation for tidyverse packages here:
+
+- [`dplyr` documentation](https://dplyr.tidyverse.org/)
+- [`tidyr` documentation](https://tidyr.tidyverse.org/)
+- [`tibble` documentation](https://tibble.tidyverse.org/)
+- [`stringr` documentation](https://stringr.tidyverse.org/)
+- [`purrr` documentation](https://purrr.tidyverse.org/)
+- [`ggplot2` documentation](https://ggplot2.tidyverse.org/)
+
+<br>
+
+| Library/Package| Piece of Code| What it's called| What it does |
```

Comment:
For organization: I think I would separate out `dplyr/tidyr/tibble` from `purrr` from `ggplot2`

Reply:
Done in https://github.com/AlexsLemonade/training-modules/pull/663/commits/024f202677e51798fa9bd81c73d6f478a52c15ad